docs/pages/numberfield.js

```diff
@@ -0,0 +1,246 @@
+// @flow strict
+import { type Node } from 'react';
+import Example from '../components/Example.js';
+import PageHeader from '../components/PageHeader.js';
+import Card from '../components/Card.js';
+import MainSection from '../components/MainSection.js';
+import Page from '../components/Page.js';
+import GeneratedPropTable from '../components/GeneratedPropTable.js';
+import docgen, { type DocGen } from '../components/docgen.js';
+
+export default function NumberFieldPage({ generatedDocGen }: {| generatedDocGen: DocGen |}): Node {
+  return (
+    <Page title="NumberField">
+      <PageHeader name="NumberField" description={generatedDocGen?.description} />
```

Comment:
can you add a defaultCode example here?

Reply:
I wasn't going to update this to the new docs style, as we can do that in the future and this is basically a copy/paste/tweak from TextField's docs